resource-watch/prep-app
# Bug: Explore export used buggy duplicated code

This PR removes duplicate code that contained bugs for the Explore export page.

## Testing instructions
1. Create a widget from the Explore page
2. Go to http://localhost:3000/export/embed/XXX where XXX is the ID of the widget

The widget should be shown exactly as in http://localhost:3000/embed/embed/XXX (replace the XXX).

## Pivotal
Part of [this task](https://www.pivotaltracker.com/story/show/159172219).
